packages/syntax-icons/src/Accent.tsx

```diff
@@ -0,0 +1,16 @@
+import { type ComponentProps, forwardRef } from "react";
+import Icon from "./Icon";
+
+const Accent = forwardRef<
```

Comment:
@somethiiing probably out of scope for this PR but would have been great to use something like [svgr](https://react-svgr.com/docs/node-api/) to automatically generate these icons based on the original SVGs (makes maintaining and adding new icons easier)